website/src/hooks/ui/useUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useRef, useState } from "react";
+
+/**
+ *  This is a custom hook that returns a ref that can be used to focus an element.
+ *
+ *   Use like:
+ *
+ *
+ *   const focusElementRef = useFocusElement();
+ *   return <div ref={focusElementRef} />
+ */
+export const useFocusElement = <T extends HTMLElement>() => {
```

Comment:
Maybe this should be `useScrollToElementOnMount`? Also can name the file according to the hook's name?

Reply:
done